src/langcheck/metrics/eval_clients/__init__.py

```diff
@@ -1,44 +1,57 @@
-from langcheck.metrics.eval_clients._base import EvalClient
+from langcheck.metrics.eval_clients._base import EvalClient, Extractor
 from langcheck.metrics.eval_clients._openai import (
     AzureOpenAIEvalClient,
+    AzureOpenAIExtractor,
     OpenAIEvalClient,
+    OpenAIExtractor,
+)
+from langcheck.metrics.eval_clients._openrouter import (
+    OpenRouterEvalClient,
+    OpenRouterExtractor,
 )
-from langcheck.metrics.eval_clients._openrouter import OpenRouterEvalClient
 
 __all__ = [
     "AzureOpenAIEvalClient",
+    "AzureOpenAIExtractor",
     "EvalClient",
+    "Extractor",
     "OpenAIEvalClient",
+    "OpenAIExtractor",
     "OpenRouterEvalClient",
+    "OpenRouterExtractor",
 ]
 
 try:
     from langcheck.metrics.eval_clients._anthropic import (
         AnthropicEvalClient,  # NOQA: F401
+        AnthropicExtractor,  # NOQA: F401
     )
 except ModuleNotFoundError:
     pass
 else:
-    __all__.append("AnthropicEvalClient")
+    __all__.extend(["AnthropicEvalClient", "AnthropicExtractor"])
 
 try:
     from langcheck.metrics.eval_clients._gemini import (
         GeminiEvalClient,  # NOQA: F401
+        GeminiExtractor,  # NOQA: F401
     )
 except ModuleNotFoundError:
     pass
 else:
-    __all__.append("GeminiEvalClient")
+    __all__.extend(["GeminiEvalClient", "GeminiExtractor"])
 
 try:
     from langcheck.metrics.eval_clients._llama import (
         LlamaEvalClient,  # NOQA: F401
+        LlamaExtractor,  # NOQA: F401
     )
     from langcheck.metrics.eval_clients._prometheus import (
         PrometheusEvalClient,  # NOQA: F401
+        StringMatchExtractor,  # NOQA: F401
```

Comment:
This class itself doesn't have any dependency issues, so we could take this out of this `try...catch` if a new file is created for this class!

Reply:
Fixed! [5b0b423](https://github.com/citadel-ai/langcheck/pull/196/commits/5b0b423a9ed680d3b6e29cad06160086618eb8ea)